環境 Python 3.11.5

- 需要有個有效的chatGPT API_KEY 
```
{
"openai_apikey":"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"
}
```
